nvtabular/ops/categorify.py

```diff
@@ -266,6 +276,8 @@ def apply_op(
                 freq_threshold=self.freq_threshold[name]
                 if isinstance(self.freq_threshold, dict)
                 else self.freq_threshold,
+                search_sorted=self.search_sorted,
+                isfreqthreshold=self.isfreqthreshold,
```

Comment:
Why do we need this parameter? Can't we just keep the logic of freq_threshold > 0 in the encode function?

Reply:
when  freq_threshold > 0 , I was getting error when `freq_threshold` is set as a dictionary, so I decided to create an extra param.

Reply:
I think we can remove the parameter. 

The _encode function doesn't take a dictionary as the freq_threshold param, and only takes integers (like we get the value from the dictionary on line 276 if its a dict)